trillian/integration/demo-script.sh

```diff
@@ -95,3 +95,25 @@ echo 'Remember to kill off all of the jobs, so their (hard-coded) ports get free
 ./trillian/integration/ct_killall.sh
 echo '...but ct_killall does not kill the hammer'
 killall -9 ct_hammer
+
+
+# Other ideas to extend a linear demo:
```

Comment:
Seems like a different PR?

Reply:
Same PR, different commit; I've adjusted the PR description.